client/socket.go

```diff
@@ -50,9 +89,23 @@ func (api *APIClient) SignSocketKey(ctx context.Context, idOrName string, in *So
 
 // Socket represents a socket in Border0 API.
 type Socket struct {
-	Name       string `json:"name"`
-	SocketID   string `json:"socket_id"`
-	SocketType string `json:"socket_type"`
+	Name                 string            `json:"name"`
+	SocketID             string            `json:"socket_id"`
+	SocketType           string            `json:"socket_type"`
+	Description          string            `json:"description,omitempty"`
+	UpstreamType         string            `json:"upstream_type,omitempty"`
+	UpstreamHTTPHostname string            `json:"upstream_http_hostname,omitempty"`
```

Comment:
This seems inconsistent, can we remove it?

Reply:
This is for HTTP socket, for example

```shell
border0 socket create \
  --type http \
  --upstream_type https \
  --upstream_http_hostname www.bbc.com \
  --name bbc 
```

I can change that later once we made changes in the api.